airflow/www/templates/airflow/log.html

```diff
@@ -0,0 +1,70 @@
+{#
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+#}
+{% extends "airflow/task_instance.html" %}
+{% block title %}Airflow - DAGs{% endblock %}
+
+{% block body %}
+{{ super() }}
+
+<div id="error" style="display: none; margin-top: 10px;" class="alert alert-danger" role="alert">
+    <span class="glyphicon glyphicon-exclamation-sign" aria-hidden="true"></span>
+    <span id="error_msg">Oops.</span>
+</div>
+<button class="btn btn-default pull-right" id="refresh_button" style="display: inline-flex;">
+    <span class="glyphicon glyphicon-refresh" aria-hidden="true" title="Refresh Logs"></span>
+</button>
+<h4>Log</h4>
+
+{% if log %}
+<pre id="log_block">{{ log }}</pre>
+<img id="loading" alt="spinner" src="{{ url_for('static', filename='loading.gif') }}"
+     style="margin: auto; display: none;">
+{% endif %}
+{% endblock %}
+
+{% block tail %}
+{{ super() }}
+<script>
+    function error(msg){
+        $('#error_msg').html(msg);
+        $('#error').show();
+        $('#loading').hide();
+        $('pre#log_block').show();
+    }
+
+    $("#refresh_button").on("click",
+        function() {
+            $("#loading").css("display", "block");
+            $('#error').hide();
+            $("pre#log_block").hide();
+            $.get(decodeURIComponent(window.location.href))
+                .done(
+                function(data) {
+                    $("#loading").hide();
+                    $("pre#log_block").html(data);
+                    $("pre#log_block").show();
+                    $('#error').hide();
```

Comment:
yeah, I started keeping that in mind only, but thought of keeping it manual for now. If we do auto refresh then we've to provide a option (checkbox) to disable that as well.